src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree-item.html

```diff
@@ -14,7 +14,14 @@
         <a class="umb-tree-item__label" ng-href="#/{{::node.routePath}}" ng-click="select(node, $event)" title="{{::node.title}}">{{node.name}}</a>
 
         <!-- NOTE: These are the 'option' elipses -->
-        <button data-element="tree-item-options" class="umb-options btn-reset sr-only sr-only--focusable sr-only--hoverable" ng-click="options(node, $event)" ng-if="::node.menuUrl" aria-label="{{optionsText}} {{node.name}}"><i></i><i></i><i></i></button>
+        <umb-button-ellipsis
+            ng-if="::node.menuUrl"
+            element="tree-item-options"
+            action="options(node, $event)"
+            text="{{optionsText}} {{node.name}}"
+            state="hidden"
+            >
+        </umb-button-ellipsis>
 
```

Comment:
@BatJan @nul800sebastiaan it seems this change cause all tree pickers to have the action button ellipsis and not only the global tree. E.g. in macro partial view picker.

**Previous**

```
<button data-element="tree-item-options" class="umb-options btn-reset sr-only sr-only--focusable sr-only--hoverable" ng-click="options(node, $event)" ng-if="::node.menuUrl" aria-label="{{optionsText}} {{node.name}}"><i></i><i></i><i></i></button>
```

![image](https://user-images.githubusercontent.com/2919859/88474740-d1483300-cf29-11ea-849b-faa0e31d0360.png)

**Now**

```
<umb-button-ellipsis
            ng-if="::node.menuUrl"
            element="tree-item-options"
            action="options(node, $event)"
            text="{{optionsText}} {{node.name}}"
            state="hidden">
</umb-button-ellipsis>
```

![image](https://user-images.githubusercontent.com/2919859/88474759-ffc60e00-cf29-11ea-834d-7f5f14ef3343.png)

Reply:
It seems to happen because `hide-options` on `umb-tree` is not respected, where it set a class `hide-options`.

https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree.html#L1

This then hide any rendered action buttons in tree:
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/less/components/tree/umb-tree.less#L217-L219

Reply:
PR submitted: https://github.com/umbraco/Umbraco-CMS/pull/8481